### DISCUSSION TITLE
2022/12/19/It-s-just-the-Beginning/

### DISCUSSION THREAD

**Discussion by giscus** [MrWillCom/MrWillCom.github.io#8] [Announcements]
# 2022/12/19/It-s-just-the-Beginning/

https://blog.mrwillcom.com/2022/12/19/It-s-just-the-Beginning/

**Comment by liyanana** [comment]
太牛了吧，初中就搞这些了

**Comment by HaoTian22** [comment]
初中就开始自己写主题了，未来可期啊
~~（到高中了还只会别人基础上魔改出来个半成品的我表示羡慕）~~

**Comment by SIPC** [comment]
膜拜大佬 orz

**Comment by BraelynLuz** [comment]
OMG, High school student. Amazing
